README.md

```diff
@@ -37,7 +37,7 @@ You can also run `yarn setup:api-data` if you just want to download data and not
 To run the website locally, use:
 
 ```shell
-gatsby develop
+yarn run develop
```

Comment:
Thanks, @codywall! Let's change this to `yarn develop` to make it even shorter :)

Reply:
Thanks @kevee! I forgot you don't need to type `run` 🤦. Added a commit to fix that.